torch/_export/converter.py

```diff
@@ -314,6 +316,68 @@ def convert_aten_div(self, node: torch._C.Node):
 
         self.convert_aten_op(node)
 
+    def convert_prim_if(self, node: torch._C.Node):
+        predicate = []
+        for input_ in node.inputs():
+            predicate.append(self.name_to_node[input_.debugName()])
+        assert len(predicate) == 1
+
+        # Get union of inputs to blocks
+        arguments = set()
+        for block in node.blocks():
+            block_args = set()
+
+            # TODO: block.inputs(), not sure what theyre used for
+
+            for block_node in block.nodes():
+                for block_node_in in block_node.inputs():
+                    if block_node_in.debugName() in self.name_to_node:
+                        block_args.add(block_node_in.debugName())
+
+            arguments.update(block_args)
+
+        # Convert blocks to subgraphs
+        subgraph_nodes = []
+        for block in node.blocks():
+            subgraph_converter = TS2GraphConverter(block, set(), set())
+            subgraph_converter.constant_map = self.constant_map
+
+            for block_arg in arguments:
+                normalized_block_arg_name = normalize_name(block_arg)
+                placeholder_node = subgraph_converter.fx_graph.placeholder(
+                    normalized_block_arg_name
+                )
+                subgraph_converter.name_to_node[block_arg] = placeholder_node
+
+            subgraph = subgraph_converter.convert()
+            subgraph_name = self.add_subgraph(subgraph)
+            subgraph_nodes.append(self.fx_graph.get_attr(subgraph_name))
+
+        assert len(subgraph_nodes) == 2
+
+        fx_block_args = [self.name_to_node[arg_name] for arg_name in arguments]
+        args = (
+            predicate[0],
+            subgraph_nodes[0],
+            subgraph_nodes[1],
+            tuple(fx_block_args),
+        )
+
+        cond_node = self.fx_graph.call_function(torch.cond, args, {})
+        getitem_node = self.fx_graph.call_function(operator.getitem, (cond_node, 0), {})
+
+        output_name = node.output().debugName()
+        self.name_to_node[output_name] = getitem_node
+
+    def convert_as_noop(self, node: torch._C.Node):
+        target = get_op_overload(node)
+        schema = target._schema
+
+        args, kwargs = self.get_args_kwargs(node, schema)
+
+        output_name = node.output().debugName()
+        self.name_to_node[output_name] = args[0]
```

Comment:
For aten::Bool, you can write it as `args[0]` here. 
Do we expect this to also work for other ops? 
If not, I would suggest specializing this function as `convert_aten_Bool`

Reply:
I think it would be somewhat common for whenever we want to convert some operator as a no-op, we can just set it as the first argument. What do you think?